### PR TITLE
Add deployment methods section to welcome page

### PR DESCRIPTION
**Pull request by rguske**
Signed-off-by: Robert Guske <rguske@vmware.com>

## Summary

This PR will add a new section to the website (Home) which highlights the two supported deployment methods for VEBA.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [x] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

Closes: #517

## Testing Verification

* Successfully verified website adjustments locally

image

## Additional Information

N/A

### CODE REVIEW

**Comment by pksrc** [comment]
Design LGTM, One suggestion that I have is to maybe use this section with a smaller version of the images. I say that because I believe that section does not have super strong CTAs and could be repurposed for this content. 
image

**Comment by rguske** [comment]
> Design LGTM, One suggestion that I have is to maybe use this section with a smaller version of the images. I say that because I believe that section does not have super strong CTAs and could be repurposed for this content. 
> 
> image
> 
> 

Thanks for your review PK. Code-wise it's fine?
I've chosen this way of highlighting the deployment options, because this topic came up several times in customer discussions and IMO our supported options should be very clear (highlighted). Also, it fits very well for the mobile version of the website which is very important IMO too.

**Comment by rguske** [comment]
@lamw @embano1 @pksrc 

Here are further alternatives according to what we've discussed on Friday. Please comment.

1. w/ section header
image

2. w/o section header
image

3. In grey w/o icons
image

**Comment by pksrc** [comment]
I like the last one - Can you remove the first two items - Explore the documentation and Get started quickly. And add a title that indicates that this section is for deployment types?

**Comment by vladi-velikov** [comment]
Like @pksrc I prefer 3. too.
Just IMO we should use either a grey area with no title or a title without a grey area. Otherwise it becomes confusing to me what kind of info this is and it's relation to the other sections (e.g. sibling or not a sibling).

**Comment by rguske** [comment]
Thanks a lot for your feedback @pksrc and @vladi-velikov 👍🏻 
I'd like to keep the icons as well and agree with your suggestions.

image

**Comment by embano1** [comment]
> I'd like to keep the icons as well

I also prefer option three (as the other gentlemen), but that option did not have the icons and IMHO it looks better w/out icons.

**Comment by rguske** [comment]
Happy?

image

**Comment by embano1** [comment]
What about option "3 In grey w/o icons" you showed above? I'm not sure featuring deployment methods on the main page is a key selling item?

**Comment by rguske** [comment]
@lamw @embano1
Heres now the final version for this part based on what we've decided yesterday. I'm happy with it 👍🏻 
@pksrc Your final review would be great. Thanks.

image

**Comment by embano1** [comment]
Can you please squash the commits?

File: docs/index.html [hunk 1]
```diff
@@ -31,10 +31,18 @@
     content: Build functions using the language of your choice
 secondary_ctas:
   cta1:
+    title: Appliance for VMware vSphere
+    url: /kb/install-knative
+    content: Deploy VEBA directly onto vSphere by using the ready-to-run appliance (OVA) method.
```
Comment (embano1) [inline docs/index.html:36]:
```suggestion
    content: Deploy VEBA directly onto vSphere by using the ready-to-run appliance (OVA.
```

**Review by embano1 — CHANGES_REQUESTED** [review]
Thx, one nit.

**Review by embano1 — APPROVED** [review]
THX!

**Pull request merged**